### PR TITLE
bradley/permissions

### PR DESCRIPTION
**Pull request by bschulz01**
Permissions for paying a numerical lab buck amount and lab hour message replaces old message

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -552,7 +553,13 @@ async def lab_hours_reminder():
             if special_hours:
                 description += f'\n**Special Lab Hours!**:\n{special_hours}'
             embed = discord.Embed(title=title, description=description, color=color)
-            await lab_channel.send(embed=embed)
+
+            if lab_channel:
+                # Send differently if plaintext or rich embed
+                if type(embed) is str:
```
Comment (ble86) [inline bot.py]: This seems unnecessary since embed is set as a discord embed a couple lines above

**Review by ble86 — COMMENTED** [review]
Made one comment, but otherwise looks good

File: bot.py [hunk 2]
```diff
@@ -747,7 +759,7 @@ async def rewards(ctx, *args):
             msg = "Available rewards```\n"
         # Get prizes for a specific category
         elif len(args) == 1:
-            rewards_dict = firebase.get_rewards(args[0])
+            rewards_dict = firebase.get_rewards(args[0].lower())
```
Comment (ble86) [inline bot.py:757]: I would suggest making this lower() command for any places where there are arguments (people's names, prizes, etc.)

**Review by ble86 — COMMENTED** [review]
Actually I made one more comment about the use of lower()

File: bot.py [hunk 3]
```diff
@@ -552,7 +553,9 @@ async def lab_hours_reminder():
             if special_hours:
                 description += f'\n**Special Lab Hours!**:\n{special_hours}'
             embed = discord.Embed(title=title, description=description, color=color)
-            await lab_channel.send(embed=embed)
+
+            if lab_channel:
```
Comment (bryanjwong) [inline bot.py]: This conditional is not necessary, on line 537 method returns if `lab_channel` is invalid

**Pull request merged**